irc.go

```diff
@@ -23,7 +23,7 @@ var (
 var (
 	ping    = regexp.MustCompile(`^PING :([^\r\n]+)\r\n$`)
 	privmsg = regexp.MustCompile(`^:([^\r\n @]+)![^\r\n @]+@[^\r\n @]+\.tmi\.twitch\.tv PRIVMSG [^\r\n ]+ :([^\r\n]+)\r\n$`)
-	youtube = regexp.MustCompile(`(?:youtube\.com/watch\?v=|youtu\.be/)([A-Za-z0-9_-]{11})`)
+	link    = regexp.MustCompile(`http[s?]:\/\/[(youtube.com\/watch\?v=|youtu.be\/|bandcamp.com\/)\S]+`)
```

Comment:
I'm not sure capturing any link starting with these prefixes is the right move. Everything else looks good.